odoo/tools/cloc.py

```diff
@@ -163,7 +163,12 @@ def count_customization(self, env):
             SELECT f.id, m.name FROM ir_model_fields AS f
                 LEFT JOIN ir_model_data AS d ON (d.res_id = f.id AND d.model = 'ir.model.fields')
                 LEFT JOIN ir_module_module AS m ON m.name = d.module
-            WHERE f.compute IS NOT null AND (m.name IS null {})
+            WHERE f.compute IS NOT null AND (
+                -- Do not count studio field
+                (m.name IS null AND (d.module != 'studio_customization' or d.module is NULL))
+                -- Unless they start with x_studio
+                OR (d.module = 'studio_customization' AND f.name ilike 'x_studio%')
```

Comment:
`_` is a wildcard in `ilike`. Escape it. The string should raw (`r"""`).
```suggestion
                OR (d.module = 'studio_customization' AND f.name ilike 'x\_studio%')
```